src/org/labkey/test/tests/SecurityTest.java

```diff
@@ -553,6 +555,23 @@ protected void clonePermissionsTest()
         assertNavTrail("Site Users", "User Details", "Permissions");
     }
 
+    protected void projectUsersTest()
+    {
+        beginAt("/project/SecurityVerifyProject/begin.view?");
+        impersonateRoles(PROJECT_ADMIN_ROLE);
+        ShowUsersPage usersPage = goToProjectUsers();
+        String userEmail = "fromprojectusers@gmail.com";
```

Comment:
Don't use real domains for test email addresses. We don't want to end up on a spam list from TeamCity spitting nonsense emails into the world. We usually use "`.test`" as it is a reserved top level domain, safe for testing.
```suggestion
        String userEmail = "fromprojectusers@security.test";
```
Also, we should delete this user during cleanup, otherwise this test would fail if you tried to run it twice. Define the email in a class constant and add it to the list of users to delete in `doCleanup` like we do with the other users (e.g. `NORMAL_USER`).

Reply:
Ah, gotcha! Done.